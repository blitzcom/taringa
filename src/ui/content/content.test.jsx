import renderer from 'react-test-renderer';

import Content from './content';

describe('ui/content', () => {
  def('children', () => <p>__CHILDREN__</p>);
  def('subject', () => renderer.create(<Content>{$children}</Content>).toJSON());

  it('renders content', () => {
    expect($subject).toMatchSnapshot();
  });
});
