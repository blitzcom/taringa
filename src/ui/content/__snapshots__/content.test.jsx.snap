// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/content renders content 1`] = `
<main
  className="content"
>
  <p>
    __CHILDREN__
  </p>
</main>
`;
